### DISCUSSION TITLE
UI优化：顶部标签高度感觉可以稍微降低一些

### DISCUSSION THREAD

**Discussion by pheotry** [ywmoyue/biliuwp-lite#864] [General]
### 问题描述

目前的高度感觉太高了
![image](https://github.com/user-attachments/assets/2a4dcd8b-ab5a-4867-8cbd-a65d718d0072)

### 描述解决方案

调成和浏览器标签的高度差不多感觉刚好

### 代码贡献

否

**Comment by ywmoyue** [comment]
目前是为了最大化窗口之后鼠标直接甩到目标标签最上方即可点击，如果调成和浏览器一样的高度就做不到这个效果

**Comment by ywmoyue** [comment]
@pheotry  这个问题先在这里讨论，确定要做之后再建issue

**Comment by ProJend** [comment]
@pheotry 你提议的高度过于紧凑，不适宜单独修改
![屏幕截图 2024-12-07 020314](https://github.com/user-attachments/assets/b807f989-8659-42df-b7c2-122765df756b)

**Reply by pheotry** [reply]
> @pheotry 你提议的高度过于紧凑，不适宜单独修改 ![屏幕截图 2024-12-07 020314](https://private-user-images.githubusercontent.com/55018532/393480359-b807f989-8659-42df-b7c2-122765df756b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM3MzM1NzEsIm5iZiI6MTczMzczMzI3MSwicGF0aCI6Ii81NTAxODUzMi8zOTM0ODAzNTktYjgwN2Y5ODktODY1OS00MmRmLWI3YzItMTIyNzY1ZGY3NTZiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA5VDA4MzQzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE3ZjdlMzNlOGI5NDdhMmY1ZmU2NzhlZDQzODQ3NTI5NDA2ZjFmMzcxODgyMWMzNjc0MzdiOTIzYzlhZjMxNDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jONnYgFX-Bjcl9UABapQam3p2A-VWAs0m6g_dGaWsKI)

这不挺好看的吗？
过于紧凑可以把内容再往下移动一些，只是标签太宽显得有点丑了

**Comment by ywmoyue** [comment]
@pheotry 4.7.8 已支持设置标签栏宽度和高度
